021 (css) passwort generator | J.H. I.B.

Lucerrion opened the pull request:
021 Passwort-Generator (CSS) https://app.asana.com/0/1205072348884926/1205158227496899/f

Lucerrion committed:
[021] Button input(+) styled

021 Passwort-Generator (CSS) https://app.asana.com/0/1205072348884926/1205158227496899/f

Lucerrion committed:
[021] version end 1.8

021 Passwort-Generator (CSS) https://app.asana.com/0/1205072348884926/1205158227496899/f

Lucerrion committed:
Merge branch 'main' of github.com:SpaTeamAcademy/Team_A

# Conflicts:
#	style.css

Lucerrion committed:
[021] added box around password

021 Passwort-Generator (CSS) https://app.asana.com/0/1205072348884926/1205158227496899/f

Lucerrion committed:
[021] Merge branch 'main' of github.com:SpaTeamAcademy/Team_A

# Conflicts:
#	style.css

Lucerrion committed:
[021] sizing changed

021 Passwort-Generator (CSS) https://app.asana.com/0/1205072348884926/1205158227496899/f

Lucerrion committed:
[021] x font weight changed

021 Passwort-Generator (CSS) https://app.asana.com/0/1205072348884926/1205158227496899/f

SpaTeamAcademy commented:
- in der Passwort-Anzeige den Text bitte vertikal mittig anzeigen.
- "press to generate" Text irreführend, da man als User denkt, man sollte in der Nähe vom Text etwas drücken. Entweder ganze Sätze schreiben (erklären, was auf dieser Seite getan werden kann) oder Erklärung ganz weglassen. Hier darauf achten, dass alle Seiten einheitlich gestaltet sind (Sprich: Wenn hier ein Erklärtext hinzukommt, dann auch bei Schere-Stein-Papier und ToDo)
- Füllfarbe der Passwort-Anzeige ist unangenehm zu betrachten, evtl. findet ihr hier noch eine bessere Lösung.

Lucerrion committed:
[021] requested changes made

021 Passwort-Generator (CSS) https://app.asana.com/0/1205072348884926/1205158227496899/f

SpaTeamAcademy commented:
- Wenn ein Passwort mit Länge >45 generiert wird, erscheint ein Scroll-Balken - ist das ein gewünschter Effekt?
- im CSS sind 2 verschiedene background-colors für #PasswordDisplay

Lucerrion committed:
[021] not needed line removed

021 Passwort-Generator (CSS) https://app.asana.com/0/1205072348884926/1205158227496899/f

Lucerrion commented:
Ich hab die nicht verwendete Farbe gelöscht. Das mit dem Scrollbalken ist gewünscht, weil wir besprochen hatten das es immer einzeilig bleiben muss.
